### PR TITLE
Fix spelling and grammar in Components

### PR DESCRIPTION
**Pull request by adrianwright109**
@mkArtakMSFT 
Using this PR in favour of previous PR #13953 which had rebase issues.

### CODE REVIEW

File: src/Components/Shared/src/ElementReferenceJsonConverter.cs [hunk 1]
```diff
@@ -30,7 +30,7 @@ public override ElementReference Read(ref Utf8JsonReader reader, Type typeToConv
                 }
                 else
                 {
-                    throw new JsonException($"Unexcepted JSON Token {reader.TokenType}.");
+                    throw new JsonException($"Unexpected JSON Token {reader.TokenType}.");
```
Comment (mkArtakMSFT) [inline src/Components/Shared/src/ElementReferenceJsonConverter.cs]: @pranavkm this is an existing exception message change. Is it safe to tweak this?

Comment (pranavkm) [reply src/Components/Shared/src/ElementReferenceJsonConverter.cs]: Yup, this should be fine

**Review by pranavkm — APPROVED** [review]
Looks great. Thanks for the PR!

**Comment by SeppPenner** [comment]
@adrianwright109 I guess, you forgot some things: https://github.com/aspnet/AspNetCore/issues/14355

**Pull request merged**